src/app/components/game/game.component.ts

```diff
@@ -66,7 +68,21 @@ export class GameComponent {
         get(this.game, 'blueTeam.spymaster') === currentUserId;
   }
 
-  endCurrentTeamsTurn() {
+  async endCurrentTeamsTurn() {
+    // If no guesses have been made, double check that they really want to end
+    // the turn Codenames rules don't allow ending a turn without making any
+    // guesses, but sometimes there's a good reason
+    if (this.currentClue.guessesMade.length === 0) {
+      const shouldEndTurn = await this.utilService.confirm(
+          'End the turn?',
+          'You are normally not allowed to end the turn without making a guess first, but you can if you must.',
+          'End the turn', 'Nevermind');
```

Comment:
What do you think about `End turn` instead of `End the turn` for both the title and the action?